site-src/geps/gep-1324.md

```diff
@@ -0,0 +1,139 @@
+    <figcaption>Decomposed view of Service</figcaption>
+  </figure>
+</div>
+<br />
+
+The "service frontend" refers to how we call a service. In a `Service`, this is an automatically allocated DNS name (`name.namespace.svc.cluster.local`) and IP address (`ClusterIP`). Not all services have frontends - for example, "headless" `Service`s.
+
+The "service backend" refers to the target of a service. In `Service`, this is `Endpoints` (or `EndpointSlice`).
+
+In the Gateway API today, `Service` is used as a "service backend". For example, in a route we would refer to `Service` as the (aptly named) `backendRef`:
+
+![image displaying how a route uses the backend function of a service](images/1324-backend-ref.png "image displaying how a route uses the backend function of the Service resource")
+
+A user connecting to a Gateway, which has a Route forwarding to "Service Backend"
+
+However, there are other ways that services _could_ be used in the API by utilizing the "service frontend".
+
+Below shows a hypothetical way to model splitting calls to the `a` service frontend to backends composed of `a-v1` and `a-v2` (see [real world example](https://linkerd.io/2.11/tasks/canary-release/)). In this case, service is used as both a frontend and a backend. The route attaches to "Service a frontend", and any calls to that will be directed to the `a-v1` and `a-v2` backends (as determined by the logic within the route.
+
+![image showing how a route could use the frontend function of the Service resource](images/1324-service-frontend.png "image showing how a route could use the frontend function of the Service resource")
+
+A user connecting to a service (frontend), which has a Route forwarding to two different services (backends)
+
+### **Producer** and **Consumer**
+
+A service **Producer** and service **Consumer** describe the two roles in a mesh.
+
+A service **producer** is someone that authors and operates a service. Some example actions they may take are deploy a `Service` and `Deployment`, setup authorization policies to limit access to their workloads, and setup routing rules to canary their workloads.
+
+A service **consumer** is someone that utilizes a service by communicating with it. For example, they may want to take actions like `curl hello-world` in their application - we could call this "consuming the `hello-world` service".
+
+Consumer and producer are personas, not concrete workloads or resources. One important aspect of these personas is that they may live in different trust boundaries (typically namespace).
+
+## Deferred Goals
+
+These are goals that we aren’t explicitly excluding, but will reconsider at a later point in time
+
+* Service mesh activation/enrollment (e.g. sidecar injection, namespace enrollment)
+* Flesh out egress gateway story
+  * How traffic gets to the gateway may start out as vendor specific, but perhaps Gateway API should standardize on exposing Routes for the egress gateway?
```

Comment:
```suggestion
* Egress (e.g. off-mesh destinations including legacy services or third-party APIs)
```
I think this is currently worded a bit confusingly, and assumes an egress gateway architecture versus allowing a sidecar egress pattern - would honestly just shorten this substantially and plan for an entire egress GEP at some future point.

Reply:
Agreed; making this change in a follow up PR